.github/workflows/Java.yml

```diff
@@ -68,10 +62,7 @@ jobs:
         run: >
           java -cp build/release/tools/jdbc/duckdb_jdbc.jar
           org.duckdb.test.TestDuckDBJDBC
-      - name: Java Example
-        shell: bash
-        run: |
-          (cd examples/jdbc; make; make maven)
```

Comment:
Why not test the examples anymore?

Reply:
Cause that test just downloads the DuckDB Jar from Maven Central, it was of limited utility